data_processing_scripts/das_protein_tools.py

```diff
@@ -0,0 +1,126 @@
+# importing necessary modules
+import protein_dict as prd
+from random import choice
```

Comment:
```suggestion
from random import choice

import data_processing_scripts.protein_dict as prd
```

Из-за неправильного импорта весь код падает при попытке запуска:). В целом это можно было бы отловить если перед сдачей прогрнать самому код (я показывал на консультации 17.10.23 как это делать).